README.md

```diff
@@ -1,123 +1,90 @@
+3. A certain subset of tickets ('assisted digital') are stored twice: they're sent once to Zendesk and once to a Google spreadsheet via the Google API. More documentation about that here: [docs/assisted_digital_feedback.md](docs/assisted_digital_feedback.md).
+4. Handles email survey signups. A banner that shows up on `gov.uk` will ask you to provide an email to signup to a survey. `feedback` will then email you a link to the survey. However, it does not collect the answers. More documentation about that here: [docs/email_survey_signups.md](docs/email_survey_signups.md) 
 
-    development> bowl feedback
+### Dependencies
 
-### Testing with real authorisation
+- [alphagov/slimmer](https://github.com/alphagov/slimmer) - provides Rack middleware for applying a standard header and footer around pages returned by a Ruby (Rack) application
+- [alphagov/govuk_frontend_toolkit](https://github.com/alphagov/govuk_frontend_toolkit) - provides Sass and Javascript files for the application's frontend
+- [alphagov/gds-api-adapters](https://github.com/alphagov/gds-api-adapters) - provides a set of API adapters to work with the GDS APIs
+- [alphagov/govuk-content-schemas](https://github.com/alphagov/govuk-content-schemas) - provides schemas and examples of the content that uses them on GOV.UK
+- [alphagov/govuk-content-schema-test-helpers](https://github.com/alphagov/govuk-content-schema-test-helpers) - provides test helpers for working with [alphagov/govuk-content-schemas](https://github.com/alphagov/govuk-content-schemas)
+- [alphagov/govuk-lint](https://github.com/alphagov/govuk-lint) - configures various linters to comply with our style guides.
+
+### Running the application
 
-In order to raise tickets in Zendesk, the `feedback` app submits data to the `support` app. As the relevant `support` app endpoints are behind `signon`, `feedback` needs a bearer token for authorisation. To get this working after an import of signon data from preview:
+To start the app using `bowler`:
+
+    bowl feedback
+
+To start the app directly:
+
+    ./startup.sh
+    
+This will start the app on port `3028`.     
+
+### Running the test suite
+
+To run unit tests, execute the following:
 
-1. Copy the token from the [support app initializer](config/initializers/support_app.rb).
+    bundle exec rake
 
-2. Start a rails console session within `signon`:
+### Testing with a mock `signon` strategy
 
-        signonotron2> bundle exec rails c
+Launch using `bowl` from the `development` directory:
 
-3. Execute the following (to update the token):
+    development> bowl feedback
 
-        u = User.find_by_email('feedback@alphagov.co.uk')
-        a = u.authorisations.first
-        a.token = "<PLACE TOKEN HERE>"
-        a.save
+### Testing with real authorisation
 
-4. To start with real authentication using `signon` and `support`:
+In order to raise tickets in Zendesk, the `feedback` app submits data to the `support` app. 
```

Comment:
This probably should read something like: 

> For the feedback that is sent to the `support` app the `feedback` app needs a bearer token for authorization because `support` is a backend app protected by `signon`.  The same is not true for the feedback that is sent to `support-api` because it's not protected by `signon`.